.gitignore

```diff
@@ -5,3 +5,4 @@
 node_modules
 aws-exports.js
 aws-exports.ts
+.npmrc
```

Comment:
Can you add a comment of why?  `.npmrc` files are valid (like `.yarnrc` here: https://github.com/aws-amplify/amplify-js/blob/main/.yarnrc) for defaults across teams.

In the future, if we want to customize this and say, "oops! this couldn't be ignored!", we won't remove it accidentally.

Reply:
Good point, I'll do that.
```suggestion
# .npmrc is used in our deployment procedure to login to npm. Ignored for security.
.npmrc
```